### ISSUE TITLE
Enum array support

### ISSUE THREAD

**Issue by bwbroersma** [PostgREST/postgrest#1353]
Enum looks nice in Swagger, enum arrays do not.
Here the changes that work for me for nice Swagger output, but it might not be correct for the OpenAPI standard:
https://github.com/openstate/postgrest/commit/74c5d9704233687efffda407abe7d5ef778b544f

**Comment by bwbroersma** [comment]
Ok the previous commit / provided solution is not the correct way to do it, this will also result in an
**Example Value**
```json
{
  "the-enum-array-field": "first-value"
}
```
were we would want more something like
```json
{
  "the-enum-array-field": ["first-value"]
}
```

**Comment by bwbroersma** [comment]
The correct Swagger to output is:
```json
{
  "format": "schema.typename[]",
  "type": "array",
  "items": {
    "type": "string",
    "enum": ["first-value", "second-value"]
  }
}
```

**Comment by kavitang-sa** [comment]
Any update on this issue? We have an enum array column and would want the above mentioned response from the OpenApi response.

**Comment by laurenceisla** [comment]
@kavitang-sa I don't think this will be fixed here (in the core repository), but it will be addressed in the [`postgrest-openapi` ](https://github.com/PostgREST/postgrest-openapi) project.

**Comment by kavitang-sa** [comment]
Thanks @laurenceisla I'll create an issue there to address this.

**Comment by kavitang-sa** [comment]
> > I'll create an issue there to address this.
> 
> No need to do that, we will track it here.

Great, I just read this after creating an issue. I'll remove it, thank you @wolfgangwalther !
